ksqldb-engine/src/test/java/io/confluent/ksql/function/udaf/offset/LatestByOffsetUdafTest.java

```diff
@@ -45,19 +51,83 @@ public void shouldInitialize() {
     assertThat(init, is(notNullValue()));
   }
 
+  @Test
+  public void shouldInitializeLatestN() {
+    // Given:
+    final Udaf<Integer, List<Struct>, List<Integer>> udaf = LatestByOffset.latestN(STRUCT_LONG, 2);
+
+    // When:
+    List<Struct> init = udaf.initialize();
+
+    // Then:
+    assertThat(init, is(notNullValue()));
+  }
+  
+  @Test
+  public void shouldThrowExceptionForInvalidN() {
+    try {
+      final Udaf<Integer, List<Struct>, List<Integer>> udaf = LatestByOffset
```

Comment:
Fix spot bugs error:

```suggestion
      LatestByOffset
```